index.js

```diff
@@ -160,21 +160,12 @@ module.exports = function(input, map) {
     loaderUtils.getOptions(webpack)
   );
 
-  var userEslintPath = userOptions.eslintPath;
-
-  var eslintPkgPath = "eslint/package.json";
-  if (userEslintPath) {
-    eslintPkgPath = userEslintPath + "/package.json";
-  }
-
-  var eslintVersion = require(eslintPkgPath).version;
-
   var config = assign(
     // loader defaults
     {
       cacheIdentifier: JSON.stringify({
         "eslint-loader": pkg.version,
-        eslint: eslintVersion
+        eslint: require(userOptions.eslintPath || "eslint").version
```

Comment:
Let's do this more safe - move this above and save in variable and put this in `try/catch` and use `eslintValue || 'unknown version'`

Reply:
Also old eslint version has `version`? Maybe we need more complex way to check this

Reply:
From version >= 3

Reply:
> Let's do this more safe - move this above and save in variable and put this in `try/catch` and use `eslintValue || 'unknown version'`

I will improve

Reply:
Looks we break old version https://github.com/webpack-contrib/eslint-loader/blob/master/package.json#L19

Reply:
Looks now

Reply:
I don't think any version of ESLint has the `version` field. It's `.CLIEngine.version` since v3, and v2 returns `undefined`.